amundsen_application/static/css/_buttons-default.scss

```diff
@@ -53,6 +60,16 @@
       background-color: $btn-default-color;
     }
 
+    &.muted {
+      border-color: $divider;
+      color: $text-secondary;
+      padding: 0 8px;
+
+      .icon {
+        background-color: $text-secondary;
+      }
+    }
+
```

Comment:
The designs added a new button type which is very similar to `btn-default` so `muted` is currently a modifier on `btn-default`.

Reply:
The difference is pretty subtle, the padding is smaller and the text is lighter.

<img width="202" alt="Screen Shot 2019-10-07 at 4 29 38 PM" src="https://user-images.githubusercontent.com/4338702/66356414-bf0bf280-e91f-11e9-8360-83df2ae15c9e.png">